src/core/utils.js

```diff
@@ -588,18 +588,18 @@ export function flatten(collector, item) {
  * @param {Element} elem element
  * @param {String} pfx prefix
  * @param {String} txt text
- * @param {Boolean} noLC
+ * @param {Boolean} noLC do not convert to lowercase
  * @returns {String} generated (or existing) id for element
  */
-export function addId(elem, pfx = "", txt = "", noLC = false) {
+export function addId(elem, pfx = "", txt = "") {
```

Comment:
any reason to trash the noLC from the args list?

Reply:
Oh, you are always making it lcase.

Reply:
I guess we should then remove the `@param` above

Reply:
Because nothing uses it. Will this cause any compat issue?

Reply:
Doing a quick check...

Reply:
hmmm... I couldn't find any in other repos, but maybe we should leave it.  It's part of public API, and seems a lot of people use `makeID` in custom plugins.